cypress/support/application-stub.tsx

```diff
@@ -0,0 +1,17 @@
+import '../../src/app/[locale]/globals.css';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+import {ReactNode} from 'react';
+
+export default function ApplicationStub({children}: {children: ReactNode}) {
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        // turns retries off
```

Comment:
Extremely minor 😅: `Turns`, uppercased? I'd like comments to follow the same grammar rules as 'normal' sentences.